packages/realm/scripts/submit-analytics.js

```diff
@@ -0,0 +1,27 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2024 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+/* eslint-env node */
+/* eslint-disable -- console */
+
+try {
+  const { submitAnalytics } = require("../dist/scripts/submit-analytics");
+  submitAnalytics().catch(console.error);
+} catch (err) {
+  console.error(err instanceof Error ? err.message : err);
```

Comment:
This only prints if loading the analytics file fails, since `submitAnalytics` returns a rejected promise if failing.